tests/integration/ambient/gateway_conformance_test.go

```diff
@@ -0,0 +1,124 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ambient
+
+import (
+	"testing"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/gateway-api/conformance/tests"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+
+	"istio.io/istio/pkg/kube"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/crd"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/scopes"
+)
+
+// GatewayConformanceInputs defines inputs to the gateway conformance test.
+// The upstream build requires using `testing.T` types, which we cannot pass using our framework.
+// To workaround this, we set up the inputs it TestMain.
+type GatewayConformanceInputs struct {
+	Client  kube.CLIClient
+	Cleanup bool
+}
+
+var gatewayConformanceInputs GatewayConformanceInputs
+
+// defined in sigs.k8s.io/gateway-api/conformance/base/manifests.yaml
+var conformanceNamespaces = []string{
+	"gateway-conformance-infra",
+	"gateway-conformance-app-backend",
+	"gateway-conformance-web-backend",
+}
+
+var skippedTests = map[string]string{
+	"HTTPRouteRedirectPath":          "spec is wrong, pending https://github.com/kubernetes-sigs/gateway-api/pull/1880",
+	"HTTPRouteRedirectHostAndStatus": "spec is wrong, pending https://github.com/kubernetes-sigs/gateway-api/pull/1880",
```

Comment:
this is in 0.7.0

Reply:
Looks like all of the skipped tests are; let me remove the first 2 for both pilot and ambient and the second for just ambient